expan/core/experiment.py

```diff
@@ -300,36 +300,36 @@ def delta(self, kpi_subset=None, variant_subset=None,
 			  assume_normal=True, percentiles=[2.5, 97.5],
 			  min_observations=20, nruns=10000, relative=False):
 		"""
-    Compute delta (with confidence bounds) on all applicable kpis,
-    and returns in the standard Results format.
-
-    Does this for all non-baseline variants.
-
-    TODO: Extend this function to metrics again with type-checking
-
-    Args:
-        kpi_subset (list): kpis for which to perfom delta. If set to
-            None all kpis are used.
-        variant_subset (list): Variants to use compare against baseline. If
-            set to None all variants are used.
-
-        assume_normal (boolean): specifies whether normal distribution
-            assumptions can be made
-        percentiles (list): list of percentile values to compute
-        min_observations (integer): minimum observations necessary. If
-            less observations are given, then NaN is returned
-        nruns (integer): number of bootstrap runs to perform if assume
-            normal is set to False.
-        relative (boolean): If relative==True, then the values will be
-            returned as distances below and above the mean, respectively,
-            rather than the	absolute values. In	this case, the interval is
-            mean-ret_val[0] to mean+ret_val[1]. This is more useful in many
-            situations because it corresponds with the sem() and std()
-            functions.
-
-    Returns:
-        Results object containing the computed deltas.
-    """
+	    Compute delta (with confidence bounds) on all applicable kpis,
+	    and returns in the standard Results format.
+
+	    Does this for all non-baseline variants.
+
+	    TODO: Extend this function to metrics again with type-checking
+
+	    Args:
+	        kpi_subset (list): kpis for which to perfom delta. If set to
+	            None all kpis are used.
+	        variant_subset (list): Variants to use compare against baseline. If
+	            set to None all variants are used.
+
+	        assume_normal (boolean): specifies whether normal distribution
+	            assumptions can be made
+	        percentiles (list): list of percentile values to compute
+	        min_observations (integer): minimum observations necessary. If
+	            less observations are given, then NaN is returned
+	        nruns (integer): number of bootstrap runs to perform if assume
+	            normal is set to False.
+	        relative (boolean): If relative==True, then the values will be
+	            returned as distances below and above the mean, respectively,
+	            rather than the	absolute values. In	this case, the interval is
+	            mean-ret_val[0] to mean+ret_val[1]. This is more useful in many
+	            situations because it corresponds with the sem() and std()
+	            functions.
+
+	    Returns:
+	        Results object containing the computed deltas.
+	    """
```

Comment:
all of this whitespace change makes it hard to review a PR. Would be better in the future to make these changes in a separate commit, and merge immediately without a PR for that, I think